docker-compose.yml

```diff
@@ -3,18 +3,26 @@ services:
   databus:
     image: "dbpedia/databus"
     ports:
-      - 3000:3000 
-      - 3001:8080 # exposes search
+      - 3000:3000 # HTTP: web ui
+      - 3001:8080 # HTTP: exposes search api
+      - 4000:4000 # HTTPS: web ui (when proxy server is enabled)
```

Comment:
I am a bit confused about the port config here.
I would assume that the following ports must be bound 0.0.0.0:80 and 0.0.0.0:443 for the proxy needed for caddy auto https 
Unfortunately I can not test this at the moment since we have no server available where these ports are not already in use. But we are working to have some. However, the question is how we handle this in compose. We would actually need something like a conditional port binding since when the proxy is deactivated. Or we comment this be default and it needs to be uncommented by the user if they want to use the proxy?